ui/src/app/subnets/views/SubnetDetails/StaticRoutes/StaticRoutes.tsx

```diff
@@ -198,6 +217,12 @@ const StaticRoutes = ({ subnetId }: Props): JSX.Element | null => {
         )}
         sortable
       />
+      {isAddStaticRouteOpen ? (
+        <AddStaticRouteForm
```

Comment:
Just for consistency the React pages tend to wrap forms below a table in a card e.g. https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/subnets/components/ReservedRanges/ReservedRanges.tsx#L281